front/src/applications/operationalStudies/components/Scenario/ScenarioContent.tsx

```diff
@@ -238,14 +238,14 @@ const ScenarioContent = ({
           )}
           {(displayTrainScheduleManagement === MANAGE_TRAIN_SCHEDULE_TYPES.add ||
             displayTrainScheduleManagement === MANAGE_TRAIN_SCHEDULE_TYPES.edit) && (
-            <div className="scenario-managetrainschedule" data-testid="manage-train-schedule">
+            <div className="scenario-managetimetableitem" data-testid="manage-timetable-item">
```

Comment:
perfect occasion to start using proper hyphenation

```suggestion
            <div className="scenario-manage-timetable-item" data-testid="manage-timetable-item">
```

Reply:
done !